Lib/idlelib/sidebar.py

```diff
@@ -454,7 +454,7 @@ def mkcmd(eventname):
         return "break"
 
     def grid(self):
-        self.canvas.grid(row=1, column=0, sticky=tk.NSEW, padx=2, pady=0)
+        self.canvas.grid(row=1, column=0, sticky=tk.NSEW, padx=0, pady=0)
```

Comment:
Padding defaults to 0; remove defaulted options.
```suggestion
        self.canvas.grid(row=1, column=0, sticky=tk.NSEW)
```